packages/sdks/web-component/src/lib/descope-wc/sdkScripts/fingerprint.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  load,
+  defaultEndpoint,
+  defaultScriptUrlPattern,
+} from '@fingerprintjs/fingerprintjs-pro';
+
+export const loadFingerprint = async (
+  initArgs: {
+    publicApiKey: string;
+    useCloudflareIntegration: boolean;
+    cloudflareEndpointUrl: string;
+    cloudflareScriptUrl: string;
+  },
+  _: { baseUrl?: string },
+  onTokenReady: (token: string) => void,
+) => {
+  try {
+    const {
+      publicApiKey,
+      useCloudflareIntegration,
+      cloudflareScriptUrl,
+      cloudflareEndpointUrl,
+    } = initArgs;
+
+    let endpoints = [];
+    if (useCloudflareIntegration && cloudflareEndpointUrl) {
+      const endpointUrl = new URL(cloudflareEndpointUrl);
+      endpoints = [endpointUrl.toString(), defaultEndpoint];
```

Comment:
I think you can put `cloudflareEndpointUrl` directly, and not do `new URL(...).toString()`